inc/admin.php

```diff
@@ -760,7 +778,7 @@ private function localize_dashboard_app() {
 		$routes = [];
 		foreach ( Optml_Rest::$rest_routes as $route_type ) {
 			foreach ( $route_type as $route => $details ) {
-				$routes[ $route ] = rest_url( OPTML_NAMESPACE . '/v1/' . $route );
```

Comment:
Here not sure why we removed the use of `rest_url` function ?

If I recall corectly it was added to mitigate some bugs.

Reply:
@GrigoreMihai We're using apiFetch which accepts the namespace/path of the route instead of the entire URL: https://github.com/Codeinwp/optimole-wp/blob/3dc4af5d83adf1f46dd54e349f62977be92dad0d/assets/src/utils/api.js#L56-L60 - it will work regardless if you're using pretty permalinks or default structure.

Reply:
You are right, thanks for the details they help.